src/convertToObject.js

```diff
@@ -3,16 +3,47 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related
+ * CSS properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceArr = sourceString.replace(/\s +|\n/g, '').split(';');
+
+  return sourceArr
+    .filter(item => item !== '')
+    .map(str => str.replace(/^:/, ''))
+    .reduce((objectWithStyle, el) => {
+      const key = el.split(':')[0];
+      const value = el.split(':')[1].replace(/^\s/g, '');
```

Comment:
don't see any sense to do `el.split(':')` two times here, you can do it once and save the result to variable